web/templates/index.html

```diff
@@ -9,7 +9,13 @@ <h1 class="text-base font-semibold leading-6 text-gray-900">Users</h1>
       <p class="mt-2 text-sm text-gray-700">A list of all the users in your account including their name, title, email and role.</p>
     </div>
     <div class="mt-4 sm:ml-16 sm:mt-0 sm:flex-none">
-      <button type="button" class="block rounded-md bg-indigo-600 px-3 py-2 text-center text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">Add user</button>
+      <form action="api/users" method="POST" class="flex">
+        <input type="text" name="username" placeholder="Username" class="block rounded-md bg-white border border-gray-300 px-3 py-2 text-sm font-semibold text-gray-900 shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500">
+        <input type="text" name="first_name" placeholder="First Name" class="block rounded-md bg-white border border-gray-300 px-3 py-2 text-sm font-semibold text-gray-900 shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500">
+        <input type="text" name="last_name" placeholder="Last Name" class="ml-2 block rounded-md bg-white border border-gray-300 px-3 py-2 text-sm font-semibold text-gray-900 shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500">
+        <input type="password" name="password" placeholder="Password" class="ml-2 block rounded-md bg-white border border-gray-300 px-3 py-2 text-sm font-semibold text-gray-900 shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500">
+        <button type="submit" class="ml-2 block rounded-md bg-indigo-600 px-3 py-2 text-center text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 focus-visible:ring-indigo-500 focus-visible:ring-offset-white">Add user</button>
+      </form>
```

Comment:
what are you thoughts on always displaying a form on index view?

Reply:
I actually think it's not the best way forward but wanted to get this out focusing on the actual functionality of creating a user and ended up leaving this in after testing out the new endpoint from the client and in postman. It would be much much better to have a new template that the "Add User" button sends the user to to then fill out a form!